test/integration/publishing_test.rb

```diff
@@ -18,11 +18,11 @@ class PublishingTest < ActiveSupport::TestCase
       # edition has been published
       public_updated_at: Time.zone.now.as_json
     )
-    request = stub_publishing_api_put_item(@presenter.base_path, expected_attributes)
+    requests = stub_publishing_api_put_content_links_and_publish(expected_attributes)
 
     perform_force_publishing_for(@draft_edition)
 
-    assert_requested request
+    requests.each { |request| assert_requested request }
```

Comment:
This occurs frequently enough that it could be extracted into a shared assertion. I looked at webmock and it wouldn't be too much of an undertaking to contribute there also, perhaps.